SUMMARY.md

```diff
@@ -271,7 +271,9 @@
     * [Filesystem](guides/hosting/infrastructure/filesystem.md)
     * [Message Queue](guides/hosting/infrastructure/message-queue.md)
     * [Reverse Http Cache](guides/hosting/infrastructure/reverse-http-cache.md)
-    * [Elasticsearch](guides/hosting/infrastructure/elasticsearch.md)
+    * [Elasticsearch](guides/hosting/infrastructure/README.md)
+      * [Elasticsearch](guides/hosting/infrastructure/elasticsearch/elasticsearch.md)
```

Comment:
The heading should be something more precise - as this is what it will show up as in the search results:

![image](https://user-images.githubusercontent.com/7780750/144218972-fcab6ae3-093a-4a7e-8315-ca4bdc9690a8.png)

Reply:
If you have a look at the other guides that we're providing, the naming scheme goes like:

 * "Add custom entity to DAL"
 * "Create a new admin module"
 * "Set up Elasticsearch" or 
 * "Debug Elasticsearch".
 
Please stick to that naming instead. Make sure to change the first heading of each article, as well as the headings in the SUMMARY.md